src/main/java/marquez/common/models/Field.java

```diff
@@ -27,18 +27,18 @@
 @Value
 public class Field {
   String name;
-  FieldType type;
```

Comment:
if there's a way to cleanly wrap/unwrap those in json I would replace the enum with a value type rather than remove FieldType here.